ztp/source-crs/ClusterLogServiceAccount.yaml

```diff
@@ -0,0 +1,56 @@
+---
```

Comment:
We need one CR per source-cr file. This will require one source CR file for each of these.

Reply:
Separated into three;
- one CR to create the SA
- one CR to bind audit logs role
- one CR to bind infrastructure logs role
- Dropped application logs binding